docs/en/observability/tutorials.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[observability-tutorials]]
+= Tutorials
```

Comment:
I'm putting this guide under tutorials for now because we're adding that container soon, and it's the best place until we build other how-to style guides.